src/pages/about.js

```diff
@@ -1,10 +1,99 @@
-import * as React from 'react';
-import Layout from '../components/Layout';
-
-const About = () => (
-  <Layout>
-    <h1>This is the about page!</h1>
-  </Layout>
-);
+import { graphql, useStaticQuery } from "gatsby";
+import * as React from "react";
+import Layout from "../components/Layout";
+import CallOut from "../components/Callout";
+import {
+  AboutDescription,
+  AboutTitle,
+  Section,
+  ReverseSection,
+  AboutImages,
+} from "../styles/about.styles";
+import { RichText } from "prismic-reactjs";
+
+import { H1, H2, P, MarginContainer } from "../styles/styles";
+
+const About = () => {
+  const aboutQuery = useStaticQuery(graphql`
+    query AboutQuery {
+      prismicAboutPage {
+        data {
+          callout_cta_label {
+            text
+          }
+          callout_heading {
+            text
+          }
+          main_heading {
+            text
+          }
+          sections {
+            image {
+              url
+            }
+            section_header {
+              text
+            }
+            section_text {
+              richText
+            }
+          }
+        }
+      }
+    }
+  `);
+
+  const {
+    prismicAboutPage: {
+      data: {
+        main_heading: mainHeading,
+        sections,
+        callout_heading: calloutHeading,
+        callout_cta_label: calloutLabel,
+      },
+    },
+  } = aboutQuery;
+
+  return (
+    <Layout>
+      <MarginContainer>
+        <AboutTitle>
+          <H1>{mainHeading.text}</H1>
+        </AboutTitle>
+        <Section>
+          <AboutDescription>
+            <H2 bold>{sections[0].section_header.text}</H2>
+            <P>
+              <RichText render={sections[0].section_text.richText} />
+            </P>
+          </AboutDescription>
+          <AboutImages>
+            <img
+              src={sections[0].image.url}
+              alt={sections[0].section_header.text}
+              width="100%"
+            />
+          </AboutImages>
+        </Section>
+        <ReverseSection>
+          <AboutImages>
+            <img
+              src={sections[1].image.url}
+              alt={sections[1].section_header.text}
+              width="100%"
+            />
+          </AboutImages>
+          <AboutDescription>
+            <H2 bold>{sections[1].section_header.text}</H2>
+            <P>
+              <RichText render={sections[1].section_text.richText} />
+            </P>
+          </AboutDescription>
+        </ReverseSection>
```

Comment:
this is making me wonder if we should change the prismic custom type. I think right now the sections field is a repeatable list, so hypothetically, more sections could be added to prismic, but they wouldn't appear on the site, which might be confusing later on to whoever is updating the site.

I think it might be a good idea to limit how many sections the page can have and just change the prismic type but I really don't know what's better